{"name": "Test configuration file corruption - Bug Discovery", "status": "passed", "description": "Bug discovery: corrupted configuration files not handled properly", "start": 1751312149577, "stop": 1751312149578, "uuid": "03e04ccb-bb95-4a38-a0aa-60f269f77b9b", "historyId": "5348e55a5657ffd0e7263d32182501d0", "testCaseId": "5348e55a5657ffd0e7263d32182501d0", "fullName": "unit.test_additional_units.TestConfiguration#test_config_file_corruption_bug", "labels": [{"name": "story", "value": "Unit Tests"}, {"name": "feature", "value": "Configuration Management"}, {"name": "severity", "value": "normal"}, {"name": "epic", "value": "OpenFilter QA"}, {"name": "tag", "value": "bug_discovery"}, {"name": "tag", "value": "pyramid_unit"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "unit"}, {"name": "suite", "value": "test_additional_units"}, {"name": "subSuite", "value": "TestConfiguration"}, {"name": "host", "value": "fealfr"}, {"name": "thread", "value": "30472-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "unit.test_additional_units"}]}